orchagent/intfsorch.cpp

```diff
@@ -652,6 +659,17 @@ void IntfsOrch::doTask(Consumer &consumer)
             }
         }
 
+        //TODO: consider to refactor for different private function
+        if (table_name == APP_SAG_TABLE_NAME)
```

Comment:
Could you update sonic-swss/doc with the new CONFIG_DB or APP_DB tables.

Reply:
Please update the HLD if you are planning to have additional APP_DB tables

Reply:
I'll update the HLD to address the APP_DB table change.